### PR TITLE
Gcw 764 add broser notification for admin app

### PR DESCRIPTION
**Pull request by trojanh**

### CODE REVIEW

File: app/routes/my_list/index.js [hunk 1]
```diff
@@ -4,8 +4,27 @@ import config from '../../config/environment';
...
+    var iDevices = [
+      'iPad Simulator',
+      'iPhone Simulator',
+      'iPod Simulator',
+      'iPad',
+      'iPhone',
+      'iPod'
+    ];
+
+    if (!!navigator.platform) {
```
Comment (Darshan-Chauhan) [inline app/routes/my_list/index.js]: DO we require double **!** operator?

File: app/components/notification-bar.js [hunk 2]
```diff
@@ -27,6 +27,25 @@ export default Ember.Component.extend({
...
+    var iDevices = [
+      'iPad Simulator',
+      'iPhone Simulator',
+      'iPod Simulator',
+      'iPad',
+      'iPhone',
+      'iPod'
+    ];
+
+    if (!!navigator.platform) {
```
Comment (Darshan-Chauhan) [inline app/components/notification-bar.js]: Do we require **!** operator?

Comment (trojanh) [inline app/components/notification-bar.js]: https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript
It nothing but forced type conversion to boolean value

**Comment by namrataukirde** [comment]
👍

**Pull request merged**